packages/@react-aria/utils/src/inertValue.ts

```diff
@@ -0,0 +1,11 @@
+import {version} from 'react';
+
+export function inertValue(value: any) {
+  const pieces = version.split('.');
+  const major = parseInt(pieces[0], 10);
+  if (major >= 19) {
+    return !!value;
+  }
+  // compatibility with React < 19
+  return value ? 'true' : undefined;
```

Comment:
Should this line actually be this?
```suggestion
  if value == undefined {
    return undefined;
  }
  return value ? 'true' : 'false';
```

Reply:
Yes, this is also how we handled it before. See https://github.com/facebook/React/issues/17157 for more details about the behavior of the `inert` attribute in React < 19.